### PR TITLE
3-get-apis finished

### PR DESCRIPTION
**Pull request by DexterGundar**

### CODE REVIEW

File: controllers/app.controllers.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { fetchTopics, fetchApi } = require("../models/app.models.js")
+const endPoints = require("../endpoints.json");
+
+exports.getTopics = (req, res, next) => {
```
Comment (B7Bprog) [inline controllers/app.controllers.js:4]: This is for the previous task and shouldn't be included in this PR.

File: controllers/app.controllers.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    .then((topics)=>{
+        res.status(200).send({ topics })
+    })
+    .catch((err)=>{
+        next(err)
+    })
+}
+
+exports.getAllApis = (req, res) =>{
+        res.status(200).send(endPoints)
```
Comment (B7Bprog) [inline controllers/app.controllers.js:16]: Endpoints could be sent back in an object on a key of 'endpoints' here.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -31,8 +32,18 @@ describe('Non existant end-points',()=>{
...
   });
+})
+
+describe('get api',()=>{
+  test("return 200 and api endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({body})=>{
+        expect(body).toEqual(endPoints)
```
Comment (B7Bprog) [inline __tests__/app.test.js:46]: Endpoints could come in on a key of 'endpoints'.

**Review by B7Bprog — APPROVED** [review]
Hey Gundars, please see my comment as a suggestion, otherwise it's good to go.

File: controllers/app.controllers.js (hunk 1 above)
Comment (DexterGundar) [reply controllers/app.controllers.js:4]: I have no idea how it got there...

**Review by B7Bprog — APPROVED** [review]
Hey Gundars, good to merge. 👍
